deploy/hypershift-obo-alertmanager-config/hypershift-obo-alertmanager-config.secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  alertmanager.yaml: Z2xvYmFsOgogIHJlc29sdmVfdGltZW91dDogNW0Kcm91dGU6CiAgcmVjZWl2ZXI6ICJudWxsIgogIHJvdXRlczoKICAtIHJlY2VpdmVyOiBvY21hZ2VudAogICAgbWF0Y2hlcnM6CiAgICAtIHNlbmRfbWFuYWdlZF9ub3RpZmljYXRpb24gPSB0cnVlCiAgICByZXBlYXRfaW50ZXJ2YWw6IDEwbQogIC0gcmVjZWl2ZXI6ICJudWxsIgogIGdyb3VwX3dhaXQ6IDMwcwogIGdyb3VwX2ludGVydmFsOiA1bQogIHJlcGVhdF9pbnRlcnZhbDogMTJoCnJlY2VpdmVyczoKLSBuYW1lOiBvY21hZ2VudAogIHdlYmhvb2tfY29uZmlnczoKICAtIHNlbmRfcmVzb2x2ZWQ6IHRydWUKICAgIHVybDogaHR0cDovL29jbS1hZ2VudC1mbGVldC5vcGVuc2hpZnQtb2NtLWFnZW50LW9wZXJhdG9yLnN2Yy5jbHVzdGVyLmxvY2FsOjgwODEvYWxlcnRtYW5hZ2VyLXJlY2VpdmVyCi0gbmFtZTogIm51bGwiCg==
```

Comment:
url: http://ocm-agent-fleet.openshift-ocm-agent-operator.svc.cluster.local:8081/alertmanager-receiver

not sure if we can use a variable here

Reply:
> url: http://ocm-agent-fleet.openshift-ocm-agent-operator.svc.cluster.local:8081/alertmanager-receiver
> 
> not sure if we can use a variable here

You mean having the variable store in a separated file in the same folder of the secret file and just defined the variable name in the secret.yaml?  Let me check if there is any existing SSS following the pattern.

Reply:
hmm, never mind, I think we hard coded the ocm-agent name here as well. https://github.com/openshift/ocm-agent/blob/master/hack/00-ocm-agent.selectorsyncset.yaml.tmpl#L55
So it should be ok.